frontend/src/stores/gameStore.ts

```diff
@@ -36,14 +49,32 @@ export const useGameStore = defineStore('games', {
 			await this.refreshAllGames();
 		},
 
-		isAvailable(): boolean {
-			const me: User = useUserStore().getMe;
-			// if (!me.isOnline)
-			// retunr false;
-			me.games.forEach((game: Game) => {
-				if (game.state === gameStates.ACTIVE) return false;
-			});
-			return true;
-		},
+		// isAvailable(): boolean {
+		// 	const me: User = useUserStore().getMe;
+		// 	if (!me.isOnline)
+		// 	   return false;
+		// 	me.games.forEach((game: Game) => {
+		// 		if (game.state === gameStates.ACTIVE) return false;
+		// 	});
+		// 	return true;
+		// },
+
+		async createGame(createdGameForm: CreateGameForm){
+			try {
+				createdGameForm.player_one = useUserStore().me.id;
+				await postRequest('games', createdGameForm);
+				await router.push('/chat'); //the router push is for later, I can imagine you want to return to your current chat @vvissche?
+				// await router.push({
+				// 	name: 'chat',
+				// 	params: { profile_id: createdGameForm.player_two},
+				// });
+				this.errors.length = 0;
```

Comment:
if this is for resetting the errors that might be from the `postRequest('games', ...)` then I'd suggest to put it above the post request. This way the errors are cleared when the user presses submit.